src/util/qhash.h

```diff
@@ -0,0 +1,5 @@
+#if QT_VERSION <= QT_VERSION_CHECK(6, 0, 0)
+#define QHASH_SEED_T uint
```

Comment:
Why not `typedef uint qhash_seed_t;`? The C preprocessor should only be used as the last resort.

I wonder why Qt doesn't provide such a typedef :roll_eyes:

Reply:
That is a better idea.

Reply:
done